README.md

```diff
@@ -8,11 +8,11 @@ The CI environment supports both ARM and Bicep and can be leveraged using GitHub
 
 <h1 style="color: steelblue;">Upcoming breaking changes</h1>
 
-In between now and the release of version `0.11.0`, the `main` branch is subject to several upcoming breaking changes that will affect all modules (e.g., the renaming of folders and files).
+The most recet release [`0.11.0`](https://github.com/Azure/ResourceModules/releases/tag/v0.11.0) of CARML is simultaneously the last in which the library (i.e., the modules) and CI environment are hosted in one location. Going forward, the maintainers of CARML will implement the remaining changes required to align the modules to the developed [Azure Verified Modules](aka.ms/avm) specifications. This will enable us to move & publish the library in the official [Public Bicep Registry](https://github.com/Azure/bicep-registry-modules).
```

Comment:
I'm still not sure about 0.11 being the last release. In addition to patch version that may be expected due to the multiple breaking changes, realistically speaking, I think it would be fair to limit to that after a module is published to PBR, its updates will be discontinued in CARML. Committing to a short timeframe for that to happen for the whole library, is only theoretical. I'd vouch for simply mentioning that the efforts of the CARML team will next be put on filling the gap towards the AVM specs and consequently publishing to PBR.

Reply:
Alrighty. Got some alternative ideas. Will update 🙂

Reply:
Updated it to a new proposal. Please check

Reply:
I like - reads well and is very clear :)